index.js

```diff
@@ -133,9 +134,33 @@ module.exports = function(content) {
         exportsString = "export default ";
 	}
 
+	var MAX_DPR = 2;
+
  	return exportsString + content.replace(/xxxHTMLLINKxxx[0-9\.]+xxx/g, function(match) {
 		if(!data[match]) return match;
-		return '" + require(' + JSON.stringify(loaderUtils.urlToRequest(data[match], root)) + ') + "';
+		if(data[match].href.indexOf('?sizes[]=') !== -1) {
+			var href = data[match].href;
+			var params = href.slice(href.indexOf('?'));
+			var hrefWithoutParams = href.slice(0, href.indexOf('?'));
+
+			var requireSizes = JSON.stringify(
+				'responsive' + params + '!' + loaderUtils.urlToRequest(hrefWithoutParams, root)
```

Comment:
This should be `responsive-loader`, because webpack 2 doesn't append `-loader` automatically